.rubocop.yml

```diff
@@ -55,3 +52,18 @@ Style/StringLiterals:
 
 Style/WordArray:
   EnforcedStyle: brackets
+
+Style/StringLiterals:
+  Enabled: false
+
+Metrics/ClassLength:
+  Enabled: false
+
+Metrics/ModuleLength:
+  Enabled: false
+
+Metrics/MethodLength:
+  Enabled: false
+
+Lint/AssignmentInCondition:
+  Enabled: false
```

Comment:
@rmosolgo I just realized `Lint/AssignmentInCondition` was already defined above.

We have some code that violates this [like assignment in `while`](https://github.com/rmosolgo/graphql-ruby/blob/2e59be6482ce94a6124ce27766ad4a7910e6c589/lib/graphql/internal_representation/rewrite.rb#L154).

Did you prefer we fix those or disable this rule?